src/components/Label/__tests__/Label-tests.jest.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, fireEvent, screen } from "@testing-library/react";
+import Label from "../Label";
+import { getTestId } from "../../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../../tests/constants";
+
+describe("Label", () => {
+  const defaultTestId = getTestId(ComponentDefaultTestId.LABEL);
+
+  it("should not call onClick callback when label doens't have an onClick callback", () => {
+    const onClick = jest.fn();
+    render(<Label text="Label" />);
+    fireEvent.click(screen.getByTestId(defaultTestId));
+    expect(onClick).not.toBeCalled();
+  });
```

Comment:
this test looks a bit redundant, you're not passing the `onClick` at all so it won't be ever called